Update "Settings" to "All IDPs" + explanation (#73783)
Azure/azure-content: articles/active-directory/external-identities/one-time-passcode.md

## Note for Azure US Government customers

> The email one-time passcode feature is disabled by default in the Azure US Government cloud and your partners will dead-end unless the feature is enabled. Unlike the commercial cloud, the Azure US Government cloud does not support your partners to redeem invitations with self-service Azure Active Directory accounts.

Reviewer: Just some suggested edits for term consistency:

```suggestion
The email one-time passcode feature is disabled by default in the Azure US Government cloud. Your partners will be unable to sign in unless this feature is enabled. Unlike the Azure public cloud, the Azure US Government cloud doesn't support redeeming invitations with self-service Azure Active Directory accounts.
```

Author: Thanks @msmimart ! Looks good to me.
